client/components/ReviewList.jsx

```diff
@@ -1,29 +1,84 @@
+/* eslint-disable no-lonely-if */
 /* eslint-disable react/forbid-prop-types */
 /* eslint-disable react/require-default-props */
 /* eslint-disable no-underscore-dangle */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import ReviewListEntry from './ReviewListEntry';
+import ReviewListPageNav from './ReviewListPageNav';
 
-const ReviewList = ({ reviews, handleClick }) => (
-  <div>
-    {reviews.map((review) => (
-      <ReviewListEntry
-        handleClick={handleClick}
-        key={review._id}
-        name={review.name}
-        rating={review.rating}
-        date={review.date}
-        review={review.review}
-        attachedPic={review.attached_pic}
-        profilePic={review.profile_pic}
-        purchasedItemName={review.purchased_item_name}
-        purchasedItemPic={review.purchased_item_pic}
-        reviewData={review}
+const ReviewList = ({ reviews, handleClick }) => {
+  const [reviewPages, setReviewPages] = useState([]);
+  const [totalPages, setTotalPages] = useState(0);
```

Comment:
You might consider using a regular variable and assigning it to reviewPages.length.  It might net you a slight performance gain.  It doesn't look like you'd ever change the totalPages value, so storing it in state might be unnecessary.